[2024-04-12T12:03:03.457+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-12T12:03:03.487+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: prioritas2.write_to_csv manual__2024-04-12T11:46:48.235270+00:00 [queued]>
[2024-04-12T12:03:03.501+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: prioritas2.write_to_csv manual__2024-04-12T11:46:48.235270+00:00 [queued]>
[2024-04-12T12:03:03.502+0000] {taskinstance.py:2303} INFO - Starting attempt 9 of 10
[2024-04-12T12:03:03.525+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): write_to_csv> on 2024-04-12 11:46:48.235270+00:00
[2024-04-12T12:03:03.537+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=638) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-12T12:03:03.539+0000] {standard_task_runner.py:63} INFO - Started process 648 to run task
[2024-04-12T12:03:03.538+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'prioritas2', 'write_to_csv', 'manual__2024-04-12T11:46:48.235270+00:00', '--job-id', '161', '--raw', '--subdir', 'DAGS_FOLDER/prioritas2_dag.py', '--cfg-path', '/tmp/tmpzr6abb2p']
[2024-04-12T12:03:03.540+0000] {standard_task_runner.py:91} INFO - Job 161: Subtask write_to_csv
[2024-04-12T12:03:03.615+0000] {task_command.py:426} INFO - Running <TaskInstance: prioritas2.write_to_csv manual__2024-04-12T11:46:48.235270+00:00 [running]> on host 805e8d5936d1
[2024-04-12T12:03:03.756+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='abdan' AIRFLOW_CTX_DAG_ID='prioritas2' AIRFLOW_CTX_TASK_ID='write_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2024-04-12T11:46:48.235270+00:00' AIRFLOW_CTX_TRY_NUMBER='9' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-12T11:46:48.235270+00:00'
[2024-04-12T12:03:03.758+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-12T12:03:03.759+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-12T12:03:03.762+0000] {taskinstance.py:2890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: write_to_csv() missing 1 required positional argument: 'data'
[2024-04-12T12:03:03.774+0000] {taskinstance.py:1205} INFO - Marking task as UP_FOR_RETRY. dag_id=prioritas2, task_id=write_to_csv, execution_date=20240412T114648, start_date=20240412T120303, end_date=20240412T120303
[2024-04-12T12:03:03.803+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 161 for task write_to_csv (write_to_csv() missing 1 required positional argument: 'data'; 648)
[2024-04-12T12:03:03.835+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-04-12T12:03:03.865+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-12T12:03:03.871+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
